pkgs/by-name/al/alioth/package.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, fetchFromGitHub, rustPlatform }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "alioth";
+  version = "0.5.0";
+
+  src = fetchFromGitHub {
+    owner = "google";
+    repo = "alioth";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-K0Hx6EJYyPJZA+FLIj44BtUuZZOqWW2DUJt1QbeZyu0=";
+  };
+
+  cargoHash = "sha256-J+1SXHQJJxT0qN/ELAvwQFnKCo13ZrQClpbfleM4PkA=";
+
+  separateDebugInfo = true;
+
+  checkFlags = [
+    # Failing test case
+    "--skip" "test_align_up_panic"
```

Comment:
Why is it failing? (There's no point running tests if we just ignore failures without a reason.)

Reply:
```
---- utils::test::test_align_up_panic stdout ----
note: test did not panic as expected
```

Reply:
The tests use `debug_assert_eq!` but in nixpkgs the checks run with the release build. I've put in a proper fix.